modules/virt-deleting-templates-from-custom-namespace.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_templates/virt-deleting-templates-from-custom-namespace.adoc
+
+[id="virt-deleting-templates-from-custom-namespace_{context}"]
+= Deleting templates from a custom namespace
+
+When you delete Red Hat templates that are deployed to a custom namespace, they are regenerated. To avoid regenerating the templates, replace the existing value for the custom namespace with a different value. This causes the templates to be deployed to the custom namespace with the new value. You can now delete the templates from the original custom namespace.
+
+.Procedure
+
+. Open the `hco` CR to edit the file.
++
+[source,terminal]
+----
+$ oc edit hco -n openshift-cnv kubevirt-hyperconverged
+----
++
+. Modify the value for the `commonTemplateNamespace` attribute by replacing it with a new value for the custom namespace.
++
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  commonTemplatesNamespace: newdefaultnamespace <1>
+----
+<1> The templates are deployed from this custom namespace instead of the original custom namespace.
++
+. View the list of Red Hat templates to confirm that these templates are now deployed to the new and custom namespace.
++
+[source,terminal]
+----
+$ oc get templates -n newdefaultnamespace
+----
++
+. Delete the templates from the custom namespace that was removed from the `hco` CR.
++
+[source,terminal]
+----
+$ oc delete templates -n customnamespace --all
```

Comment:
avoid using `--all`, add single use case

Reply:
If --all is removed, which syntax is substituted here instead of --all?

Reply:
`--all` is okay here in my opinion.

Reply:
Would suggest `oc delete templates -n customnamespace {templatename}`.
I try to avoid using `--all` for any action that can be destructive.

Reply:
Updated the content.